kivy/core/video/video_ffpyplayer.py

```diff
@@ -109,6 +109,7 @@ def __init__(self, **kwargs):
         self._next_frame = None
         self._seek_queue = []
         self._ffplayer_need_quit = False
+        self._ev_trigger = Event()
```

Comment:
I think _ev_trigger is not a very good name for the purpose of the event

Reply:
I did rename the variable